website/static/css/site.css

```diff
@@ -2109,3 +2109,46 @@ span#profileFullname{
 .modal-backdrop {
     position: fixed;
 }
+
+/* Treebeard in Configure Notifications, project settings page */
+#notificationSettings #tb-tbody {
```

Comment:
There is a lot of duplicated code here. Also, these rules are rather brittle--if we change any of these CSS IDs, the UI breaks.

Could you apply a helper class instead, like `osf-treebeard-minimal`, then refactor this CSS?

```CSS
.osf-treebeard-minimal {
    border: none;
}

.osf-treebeard-minimal .tb-head {
    display: none;
}

...
```